package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "devDependencies": {
+    "jest": "^29.3.1"
+  },
+  "scripts": {
+    "test": "jest"
```

Comment:
"test": "node --experimental-vm-modules node_modules/jest/bin/jest.js"

Reply:
Got it. Completed the test cases